File: Linear/Static/Array/03_Deleting/End/Custom/Deleting.c
#include <stdio.h>

void deleteAtEnd(int arr[], int n) {
    // Reduce the array size by 1.
}

int main() {
    int arr[] = {10, 20, 30, 40};
    int n = sizeof(arr) / sizeof(arr[0]);

    printf("Array before deletion:\n");
    for (int i = 0; i < n; i++) {
        printf("%d ", arr[i]);
    }

    deleteAtEnd(arr, n);
    n--;

    printf("\nArray after deletion:\n");
    for (int i = 0; i < n; i++) {
        printf("%d ", arr[i]);
    }
    
    return 0;
}

File: Linear/Static/Array/03_Deleting/End/Custom/Deleting.py
def deleteAtEnd(arr):
    # Reduce the array size by 1.
    arr.pop()

    return arr

arr = [10, 20, 30, 40]

print("Array before deletion")
for i in range(len(arr)):
    print(arr[i], end=" ")

deleteAtEnd(arr)

print("\nArray after deletion")
for i in range(len(arr)):
    print(arr[i], end=" ")